generators/cloudfoundry/index.js

```diff
@@ -46,38 +46,38 @@ module.exports = CloudFoundryGenerator.extend({
             message: 'Name to deploy as:',
             default: this.baseName
         },
-        {
-            type: 'list',
-            name: 'cloudfoundryProfile',
-            message: 'Which profile would you like to use?',
-            choices: [
-                {
-                    value: 'dev',
-                    name: 'dev'
-                },
-                {
-                    value: 'prod',
-                    name: 'prod'
-                }
-            ],
-            default: 0
-        },
-        {
-            when: function(response) {
-                return databaseType != 'no';
+            {
```

Comment:
nope I use Atom for generator stuff. Anyways IDE has edge cases and this
could be one of them, so just edit it manually

Thanks & Regards,
Deepu

On Fri, Mar 4, 2016 at 9:54 AM, Vivek <notifications@github.com> wrote:

> In generators/cloudfoundry/index.js
> <https://github.com/jhipster/generator-jhipster/pull/3068#discussion_r54980776>
> :
>
> > -            choices: [
> > -                {
> > -                    value: 'dev',
> > -                    name: 'dev'
> > -                },
> > -                {
> > -                    value: 'prod',
> > -                    name: 'prod'
> > -                }
> > -            ],
> > -            default: 0
> > -        },
> > -        {
> > -            when: function(response) {
> > -                return databaseType != 'no';
> > +            {
>
> I double checked - the IntelliJ Idea IDE uses the .editorconfig from our
> repository to format the code like you saw in the PR.
>
> Could you please try and see if your IDE (I'm hoping you use Idea too)
> does anything differently ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/generator-jhipster/pull/3068/files#r54980776>
> .
>